### PR TITLE
Updating 3.2 release information

### PR DESCRIPTION
**Pull request by gschueler**

### CODE REVIEW

**Comment by jplassnibatt** [comment]
DEB and RPM upgrade from 3.0.x to 3.2rc4

```
[2019-12-13 12:36:41.755]  WARN AnnotationConfigEmbeddedWebApplicationContext --- [           main] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'requestMappingHandlerMapping' defined in org.springframework.web.servlet.config.annotation.DelegatingWebMvcConfiguration: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'grailsInterceptorMappedInterceptor': Cannot create inner bean '(inner bean)#449f9628' of type [org.grails.plugins.web.interceptors.GrailsInterceptorHandlerInterceptorAdapter] while setting constructor argument; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name '(inner bean)#449f9628': Unsatisfied dependency expressed through method 'setInterceptors' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'pluginsCtlInterceptor': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'uiPluginService': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'pluginService': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'frameworkService': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'authorizationService': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'configStorageService': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'rundeckConfigStorageTreeFactory': Cannot resolve reference to bean 'storagePluginProviderService' while setting bean property 'storagePluginProviderService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'storagePluginProviderService': Cannot resolve reference to bean 'rundeckFramework' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'frameworkFactory': Cannot resolve reference to bean 'frameworkFilesystem' while setting bean property 'frameworkFilesystem'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'frameworkFilesystem': Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.dtolabs.rundeck.core.common.FilesystemFramework]: Factory method 'createFilesystemFramework' threw exception; nested exception is java.lang.IllegalArgumentException: project base directory could not be created. /var/rundeck/projects
[2019-12-13 12:36:41.796] ERROR SpringApplication --- [           main] Application startup failed
```
This is because the 3.0.x has this line in framework:
framework.projects.dir=/var/rundeck/projects
But 3.2:
framework.projects.dir=/var/lib/rundeck/projects

So, if before the upgrade the /var/rundeck/projects is NOT empty, 3.2 will start properly, but if /var/rundeck/projects is empty, it will be deleted and 3.2 won't start until you modify the proper line in framework.properties to be /var/lib/rundeck/projects.

**Comment by gschueler** [comment]
@jplassnibatt is this an issue upgrading from 3.0 to 3.1 as well?

File: docs/upgrading/upgrading-to-rundeck-3.2.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
...work.beans.factory.BeanCreationException: Error creating bean with name 'rundeckConfigStorageTreeFactory': Cannot resolve reference to bean 'storagePluginProviderService' while setting bean property 'storagePluginProviderService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'storagePluginProviderService': Cannot resolve reference to bean 'rundeckFramework' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'frameworkFactory': Cannot resolve reference to bean 'frameworkFilesystem' while setting bean property 'frameworkFilesystem'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'frameworkFilesystem': Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.dtolabs.rundeck.core.common.FilesystemFramework]: Factory method 'createFilesystemFramework' threw exception; nested exception is java.lang.IllegalArgumentException: project base directory could not be created. /var/rundeck/projects
```
Comment (ProTip) [inline docs/upgrading/upgrading-to-rundeck-3.2.md]: This may be TMI

**Comment by ProTip** [comment]
@jplassnibatt Is this change highlighted by

**Comment by jplassnibatt** [comment]
> @jplassnibatt is this an issue upgrading from 3.0 to 3.1 as well?

yes @gschueler 3.1 was the first version including that path change.

File: docs/upgrading/upgrading-to-rundeck-3.2.md (hunk 1 above)
Comment (gschueler) [reply docs/upgrading/upgrading-to-rundeck-3.2.md]: yes...it can be curtailed i think

File: docs/history/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 ## Upgrading
-See the upgrade documentation [here](https://docs.rundeck.com/3.1.0-rc2/upgrading/upgrade-to-rundeck-3.1.html).
+
+See [Upgrading to Rundeck 3.2](/upgrading/upgrading-to-rundeck-3.2.md).
 
 ## Enhancements
 
-### Webhooks (incubating)
+### Webhooks 
 Much anticipated and maximally useful.. Webhooks have landed in Rundeck! This new incubating feature
```
Comment (jessemarple) [inline docs/history/README.md]: Remove second "." in sentence.

**Review by jessemarple — COMMENTED** [review]
Just one comment.

File: docs/history/README.md (hunk 2 above)
Comment (gschueler) [reply docs/history/README.md]: I think i will rephrase most of that due to no longer being incubating

**Review by jessemarple — APPROVED** [review]

**Pull request merged**